<launch>

<node name="robot_pose_ekf" pkg="robot_pose_ekf" exec="robot_pose_ekf" output="screen">
  <param name="output_frame" value="odom_combined"/>
  <param name="base_footprint_frame" value="base_footprint"/>
  <param name="freq" value="30.0"/>
  <param name="sensor_timeout" value="1.0"/>  
  <param name="odom_used" value="true"/>
  <param name="imu_used" value="true"/>
  <param name="vo_used" value="true"/>

  <remap from="odom" to="base_odometry/odom" />
  <remap from="imu_data" to="torso_lift_imu/data" />
</node>

</launch>

